src/main/java/com/brew/oauth20/server/controller/AuthorizeController.java

```diff
@@ -116,7 +143,7 @@ private ResponseEntity<String> authorize(AuthorizeRequestModel authorizeRequest,
             if (Boolean.TRUE.equals(consentRequired(authorizeRequest, clientUser)))
                 return redirectToConsent(parameters);
 
-            return redirectToRedirectUri(authorizeRequest, parameters, clientUser);
+            return redirectToRedirectUri(authorizeRequest, redirectUriParameters, clientUser);
         } catch (OAuthException e) {
             logger.error(e.getMessage(), e);
```

Comment:
Directly logging the exception message can potentially log sensitive information.

it is a general issue in project. can be interpreted as general issue.

Reply:
OAuthExceptions don't have any sensetive information since they only have OAuthError enum values.